rnbv.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const chalk = require('chalk');
+const prompts = require('prompts');
+const semver = require('semver');
+
+const configMap = new Map([
+  ['androidPath', './android/app/build.gradle'],
+  ['iosPath', './ios/ReactNativeBase/Info.plist'],
```

Comment:
The iOS directory could change when the app is renamed. After renaming the app it changes the location for every native iOS file to use the new app name as the new project folder.

Reply:
I added the `rnbv` file to the `filesToPatch` so the renames changes the value